app/api/routes.py

```diff
@@ -204,6 +211,34 @@ def get_resources():
     return standardize_response(payload=dict(data=resource_list, **pagination_details))
 
 
+def search_results():
+    term = request.args.get('q', '', str)
+    page = request.args.get('page', 0, int)
+    page_size = request.args.get('page_size', Config.RESOURCE_PAGINATOR.per_page, int)
+    query = Resource.query
+
+    search_result = index.search(f'{term}', {
+        'page': page,
+        'hitsPerPage': page_size
+    })
+
+    if page >= int(search_result['nbPages']):
+        return redirect('/404')
+
+    object_ids = [result['objectID'] for result in search_result['hits']]
+    results = [r.serialize for r in query.filter(Resource.id.in_(object_ids))]
```

Comment:
I'm fine merging this for now to unblock development on the front end, but we should refactor this to just send the results from Algolia rather than querying the database for all the search results.